digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -640,6 +650,37 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
     orderedTargets
   }
 
+  def recalculateRoadAddresses(roadNumber: Int) = {
```

Comment:
Yes, it should be changed since i already know which roadPart i want to recalculate. No need to check all existing ones